src/examples/completeExample.json

```diff
@@ -0,0 +1,23 @@
+{
+  "nodeType": "formulas",
+  "forest": [
+    {
+      "nodeType": "formulas",
+      "forest": [],
```

Comment:
After the child node, there should be a special `FinishedNode`, see the CarnapAPI file for the format.
(You can put a placeholder for the "rule" field for now.)

Reply:
`forest: OutputNode[] | [FinishedNode] | [ContradictionNode]`

I was a little confused by this, I thought it would mean that the 'forest' would be either an OutputNode or FinishedNode or ContradictionNode?

Reply:
`T[]` means an array whose members have type T.
`[T]` means an array containing exactly one T.

So, `OutputNode` covers the case where the branch continues, splits, or is open at the bottom. We have only a single node in the forest when it's one of the special nodes to mark it as closed with an X or finished with an O.

It's not quite how I'd have chosen to do it, but it's to make it easier on the Carnap backend.

Reply:
Reference for TS array notation:
https://2ality.com/2018/04/type-notation-typescript.html#array-types